packages/node_modules/nav-frontend-skjema/src/fnr-input.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import Input, { NavFrontendInputProps } from './input';
+import 'nav-frontend-skjema-style';
+
+export interface FnrInputProps extends NavFrontendInputProps {
+    onValidate: (isValid: boolean) => void;
+}
+
+function isBirthNumFormat(value) {
```

Comment:
Kan vel bare bruke funksjonen som allerede eksisterer i nav-frontend-js-utils?
Og hva gjør vi med dnr og evt andre formater? :/

Reply:
Ser ut som vi ikke her noe lignende i `nav-frontend-js-utils` i master

Reply:
Men vi burde helt sikkert støtte D-nummer osv ja. Visste ikke at det var en greie jeg. Finnes det ikke noe eksisterende bibliotek for sånt? Hadde Bekk noe greier her? 🤔

Reply:
Mener vi hadde en validerings-funksjon for fnr i nav-frontend jeg. Men mulig jeg husker feil. :/
Bekk har noen greier for validering av fnr/dnr ja, men det er skrevet i java.
En mulighet er ta denne; https://github.com/navikt/internarbeidsflatedecorator/blob/master/v2/src/utils/fnr-utils.ts
Den brukes for å sjekke fnr når man skal søke opp folk i modia-verden